### PR TITLE
Push WinoBias and StereoSet experiment code and result

### PR DESCRIPTION
**Pull request by gokejian**
Push WinoBias and StereoSet experiment code and result. The commit includes the PROCESSED datasets that can be fed into /scripts/metric.py . Also the code metric.py was modified to support scientific models and the new data format. The .slurm files are the sbatch submission files. The outputs are in the respective slurm outpur folders.

### CODE REVIEW

File: wino_stereo/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# FAQ Page
```
Comment (Chu-Wx) [inline wino_stereo/README.md:1]: looks good

File: wino_stereo/scripts/run_stereoset.slurm [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (Chu-Wx) [inline wino_stereo/scripts/run_stereoset.slurm:1]: nice

File: wino_stereo/scripts/run_winobias.slurm [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (Chu-Wx) [inline wino_stereo/scripts/run_winobias.slurm:1]: decent

File: wino_stereo/stereo_slurm_output/5800956_stereoset_all.out [hunk 4]
```diff
@@ -0,0 +1,55 @@
+Evaluating:
```
Comment (Chu-Wx) [inline wino_stereo/stereo_slurm_output/5800956_stereoset_all.out:1]: NB

File: wino_stereo/wino_slurm_output/5799851_winobias_all.out [hunk 5]
```diff
@@ -0,0 +1,110 @@
+Evaluating:
```
Comment (Chu-Wx) [inline wino_stereo/wino_slurm_output/5799851_winobias_all.out:1]: very good

**Review by L1chtyan — APPROVED** [review]

**Pull request merged**